target/linux/ath79/dts/qca9533_yuncore_cpe830.dts

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+       compatible = "yuncore,cpe830", "qca,qca9533";
+       model = "YunCore/KuWfi CPE830(D)";
+
+       chosen {
+               bootargs = "console=ttyS0,115200";
+       };
+
+       keys {
+               compatible = "gpio-keys";
+
+               reset {
+                       label = "reset";
+                       linux,code = <KEY_RESTART>;
+                       gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+               };
+       };
+
+       aliases {
+               label-mac-device = &wmac;
```

Comment:
I guess it won't work if we don't add MAC address NVMEM cell to `wmac` node.

Reply:
added the mac address